functions/__fzf_preview_file.fish

```diff
@@ -1,19 +1,33 @@
 # helper function for __fzf_search_current_dir
 function __fzf_preview_file --description "Prints a preview for the given file based on its file type." --argument-names file_path
-    set file_type (file --brief -i "$file_path")
```

Comment:
main change: switching from using file to get file type to using ls to get file type symbol